fhirr4/pom.xml

```diff
@@ -35,11 +35,10 @@
     <properties>
         <maven.compiler.source>11</maven.compiler.source>
         <maven.compiler.target>11</maven.compiler.target>
-        <ballerina.version>2201.6.0</ballerina.version>
+        <ballerina.version>2201.7.0</ballerina.version>
         <gson.version>2.8.6</gson.version>
         <r4.version>4.0.0</r4.version>
```

Comment:
what do you mean by r4 version? should It be fhir release version or something? right?

Reply:
It means the health.fhir.r4 dependency used for this package - https://github.com/ballerina-platform/module-ballerinax-health.fhir.r4/blob/fec8959bb9b3f058c1e5d808110ff0c574db4b52/fhirr4/ballerina/src/main/resources/fhirservice/Ballerina.toml#L16